docs/core/tools/dotnet.md

```diff
@@ -118,4 +118,6 @@ Specifies the location of the servicing index to use by the shared host when loa
 
 `DOTNET_CLI_TELEMETRY_OPTOUT`
 
-Specifies whether data about the .NET Core tools usage is collected and sent to Microsoft. `true` to opt-out of the telemetry feature (values true, 1 or yes accepted); otherwise, `false` (values false, 0 or no accepted). If not set, it defaults to `false`, that is, the telemetry feature is on.
\ No newline at end of file
+Specifies whether data about the .NET Core tools usage is collected and sent to Microsoft. `true` to opt-out of the telemetry feature (values true, 1 or yes accepted); otherwise, `false` (values false, 0 or no accepted). If not set, it defaults to `false`, that is, the telemetry feature is on.
+
+[dotnet-vstest](dotnet-vstest.md)
```

Comment:
this link is somewhat lost in here. perhaps you should add it as text only on your other PR and once you merge that one, we add the link?